### PR TITLE
Add Accordion styleOverride and Story

### PR DESCRIPTION
**Pull request by NataliaVizintini**
Screenshot 2023-05-10 at 11 36 53

### CODE REVIEW

File: packages/admin/admin-theme/src/componentsTheme/MuiAccordion.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            },
+            [`& .${accordionSummaryClasses.expandIconWrapper}.${accordionSummaryClasses.expanded}`]: {
+                transform: "rotate(90deg)",
+            },
+            [`& .${accordionSummaryClasses.root}`]: {
+                display: "flex",
+                flexDirection: "row-reverse",
+                padding: "20px",
+                gap: "20px",
+                height: "86px",
```
Comment (jamesricky) [inline packages/admin/admin-theme/src/componentsTheme/MuiAccordion.ts:31]: A fixed height-value will cause the text from overflowing when it requires multiple lines. 

image

File: .changeset/pretty-owls-pay.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@comet/admin-theme": patch
```
Comment (jamesricky) [inline .changeset/pretty-owls-pay.md:2]: I think this should be at least "minor", as this adds new styling and is not just a bugfix.

**Review by jamesricky — COMMENTED** [review]
`Accordion` is just a component we can use from MUI.  
I don't think this requires documentation or stories in comet.

File: .changeset/pretty-owls-pay.md (hunk 2 above)
Comment (jamesricky) [inline .changeset/pretty-owls-pay.md:2]: I think this should be at least "minor", as this adds new styling and is not just a bugfix.  

```suggestion
"@comet/admin-theme": minor
```

File: packages/admin/admin-theme/src/componentsTheme/MuiAccordion.ts (hunk 1 above)
Comment (NataliaVizintini) [reply packages/admin/admin-theme/src/componentsTheme/MuiAccordion.ts]: Good point, I removed the value

File: packages/admin/admin-theme/src/componentsTheme/MuiAccordion.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+            "&:not(:last-child)": {
+                marginBottom: "20px",
+            },
+            "&:before": {
+                display: "none",
+            },
+            [`& .${accordionSummaryClasses.expandIconWrapper}.${accordionSummaryClasses.expanded}`]: {
+                transform: "rotate(90deg)",
+            },
+            [`& .${accordionSummaryClasses.root}`]: {
```
Comment (johnnyomair) [inline packages/admin/admin-theme/src/componentsTheme/MuiAccordion.ts:26]: @jamesricky Shouldn't summary and details styles be styled in their respective components themes?

**Comment by johnnyomair** [comment]
> `Accordion` is just a component we can use from MUI. I don't think this requires documentation or stories in comet.

There should be a code example that shows how to achieve our "default" Accordion use case (title, support text, end adornment). Would you prefer only having a story for it, and no docs?

File: packages/admin/admin-theme/src/componentsTheme/MuiAccordion.ts (hunk 3 above)
Comment (jamesricky) [reply packages/admin/admin-theme/src/componentsTheme/MuiAccordion.ts:26]: Yes, good point 👍🏼

**Comment by jamesricky** [comment]
> > `Accordion` is just a component we can use from MUI. I don't think this requires documentation or stories in comet.
> 
> There should be a code example that shows how to achieve our "default" Accordion use case (title, support text, end adornment). Would you prefer only having a story for it, and no docs?

Yes, if there is a specific way we are supposed to use MUI's accordion in comet-projects, it might make sense to document this under [Best Practices](https://comet-admin.netlify.app/?path=/story/docs-best-practices-overview--page) or in the [Comet Docs](https://docs.comet-dxp.com/docs/overview/).

I don't think it should be included in the component-docs, as the component is already documented by MUI and when you work with comet you generally know to use MUI and it's docs. 

On the other hand, if we create a new component for the `Accordion`, like in https://github.com/vivid-planet/comet/pull/1082 it makes sense to have our own component docs.

**Comment by johnnyomair** [comment]
Closed in favor of #1082

**Pull request closed without merging**